workos/resources/events.py

```diff
@@ -39,7 +68,66 @@ class DirectoryDeletedEvent(EventModel[Literal["dsync.deleted"], DirectoryPayloa
     event: Literal["dsync.deleted"]
 
 
+class DirectoryGroupCreatedEvent(
+    EventModel[Literal["dsync.group.created"], DirectoryGroup]
+):
+    event: Literal["dsync.group.created"]
```

Comment:
When I set `pylance` to standard type checking rather than basic, it doesn't like this property override because it doesn't match the type on `EventModel`.

I'm going to ignore it for now, but it may rear its head again when I test what happens when an unrecognized event type gets parsed.